modules/widgets/social-media-icons.php

```diff
@@ -325,6 +325,17 @@ public function remove_username( $val ) {
  * @return void
  */
 function wpcom_social_media_icons_widget_load_widget() {
-	register_widget( 'wpcom_social_media_icons_widget' );
+	$transient = 'wpcom_social_media_icons_widget::is_active';
+	$has_widget = get_transient( $transient );
+
+	if ( false === $has_widget ) {
+		$has_widget = (int) ! empty( is_active_widget( false, false, 'wpcom_social_media_icons_widget', false ) );
```

Comment:
This is failing our PHP 5.3 linter because [arbitrary expressions](http://php.net/manual/en/migration55.new-features.php#migration55.new-features.empty) for `empty()` were introduced in 5.5